### ISSUE TITLE
Layout issues when there are multiple extruders

### ISSUE THREAD

**Issue by alfrix** [jordanruthe/KlipperScreen#262]
When not heating, it's fine
![2021-09-11-115621_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952109-40a32aae-2e10-4896-9e45-6698377bf347.png)

But the label gets too long quickly 
![2021-09-11-115635_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952112-31a12ad8-f45e-48c0-b175-e22e3815c2d5.png)
![2021-09-11-115809_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952113-4a4eb6f2-17bd-4bd4-9a80-cfb8e4a1e6df.png)
![2021-09-11-115844_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952114-ff61cea4-95c3-4460-ae90-02d02b5f39c0.png)
![2021-09-11-115847_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952116-7626b01a-db24-4870-bb22-4f602fd01b9e.png)

long temp sensor names can cause this too
![2021-09-11-120248_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952200-bc5801e4-4175-4e7c-b965-d24d135079ad.png)

basically 3 columns is just too much for this resolution (480x320)

potential fixes: change the temp label  from 
100 -> 230 C 
to:
230 C
^100^

Tried:
![2021-09-11-121019_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952563-80f1774f-20bf-4c82-a6cb-ab9025a9a466.png)
![2021-09-11-121058_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952565-378df8ba-5542-4822-a62e-c53b9629811b.png)

2 rows can work
![2021-09-11-121126_480x320_scrot](https://user-images.githubusercontent.com/1247237/132952567-4b8d7895-9e6a-4871-97a9-9935f538cc58.png)

for long label ellipsize (set max_chars if > 4 items and resolution is low)

**Comment by jordanruthe** [comment]
I do want to convert the front page to more of a table setup like fluidd or mainsail. I'll try to get to this in a few days

**Comment by jordanruthe** [comment]
PR #287 will fix this when it's merged in a few days.
